### ISSUE TITLE
Updating citation information

### ISSUE THREAD

**Issue by ChrisRackauckas** [JuliaDiffEq/DifferentialEquations.jl#273]
Oh boy oh boy. We have slacked a little bit on citation information. It should all be here:

http://juliadiffeq.org/citing.html
https://github.com/JuliaDiffEq/juliadiffeq.github.io/blob/master/citing.md

I want to make a Zotero group so that way people can easily pull all of the citations as well. First priority though is citing all of the algorithms we've implemented recently.

Looking at the citations, it looks like these releases didn't get proper additions:

http://juliadiffeq.org/2017/09/09/StiffDDESDE.html
http://juliadiffeq.org/2018/01/15/Sundials.html
http://juliadiffeq.org/2018/01/24/Parameters.html
http://juliadiffeq.org/2018/02/17/Reactions.html
http://juliadiffeq.org/2018/03/31/AdaptiveLowSDE.html
http://juliadiffeq.org/2018/04/09/AutoSwitch.html

Help me dig up the right texts to make this complete.

@onoderat 
@MSeeker1340 
@YingboMa 
@Gaussia 
@Vaibhavdixit02 
@sipah00

**Comment by ChrisRackauckas** [comment]
We should populate https://www.zotero.org/groups/2163651/juliadiffeq

**Comment by Gaussia** [comment]
I would be happy to help by providing something useful, but I am uncertain of exactly what.

If you are interested in citing work which the the Reaction tool depends on there are some papers in the 19th century which introduces Mass Kinetics.(..) Then there is the Gillespie papers, most notably from 2000 on the Chemical Langevin equations. If you know what you have in mind I could compile a (appropriately comprehensive) list.

If you mean the reverse I do not think there is something special for citing that tool (I guess one can always mention its existence in the DiffEq repository. I am uncertain what is the praxis is here). If something more formal is useful I guess something could be made.

**Comment by ChrisRackauckas** [comment]
>If you are interested in citing work which the the Reaction tool depends on there are some papers in the 19th century which introduces Mass Kinetics.(..) Then there is the Gillespie papers, most notably from 2000 on the Chemical Langevin equations. If you know what you have in mind I could compile a (appropriately comprehensive) list.

Yes. Basically, we should be able to put a section on that website that says `DiffEqBiological.jl` and `@reaction_network` and cite the work that is based on so that way users can follow the sources. That would be the Gillespie papers. If we want to put any more that's fine too. It should really just cover the methods we directly used.

**Comment by Gaussia** [comment]
There is the Gillespie papers:

Gillespie, D. T. (1977). Exact stochastic simulation of coupled chemical reactions. Journal of Physical Chemistry, 81(25), 2340–2361. https://doi.org/10.1021/j100540a008
[Gillespie_1977.pdf](https://github.com/JuliaDiffEq/DifferentialEquations.jl/files/1906969/Gillespie_1977.pdf)
Gillespie, D. T. (2000). Chemical Langevin equation. Journal of Chemical Physics, 113(1), 297–306. https://doi.org/10.1063/1.481811
[Gillespie_2000.pdf](https://github.com/JuliaDiffEq/DifferentialEquations.jl/files/1906968/Gillespie_2000.pdf)

You could also include the original work on Mass Action. A bit silly but if they ever are to get cited these days it would probably be in situations like this.

Waage, P.; Guldberg, C.M. (1864). Studier over Affiniteten. Forhandlinger i Videnskabs-selskabet i Christiania: 35–45.
Waage, P. (1864). Forsøg til Bestemmelse af Lovene for Affiniteten. Forhandlinger i Videnskabs-selskabet i Christiania: 92–94.
Guldberg, C.M. (1864). Foredrag om Lovene for Affiniteten, specielt Tidens Indflydelse paa de kemiske Processer. Forhandlinger i Videnskabs-selskabet i Christiania: 111–120.

I managed to get a PDF of the publication as well. However condiering that it is 100MB and in old Danish I think that would be to go to far...
